test/specs/chrome.js

```diff
@@ -15,9 +15,6 @@ import { accessibilityScenarios } from './scenarios/accessibility'
 describe('Happy Paths on Chrome', () => {
   // Multiple language scenarios
   fullTestCoverageLanguages.forEach((lang) => {
-    if (lang !== 'en_US') {
-      return
-    }
```

Comment:
Did you intended to run all happy path scenarios for all languages?

Reply:
It was a mistake by me since (I can't remember). Fixed it in #1264 but still missed it here

Reply:
It was a mistake by me since (I can't remember). Those happy path scenarios should run for `fullTestCoverageLanguages` (which are `en_US` and `es_ES`). At some point in the past when I attempted to fix the UI tests, I enabled them for `en_US` only and forgot to revert it back.